src/main/c/fields.h

```diff
@@ -0,0 +1,19 @@
+/*
+  Copyright (c) 2019 Criteo
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+extern jfieldID field_Max1Result_out;
+extern jfieldID field_Max1Result_x;
+extern jfieldID field_Max1Result_y;
```

Comment:
I think you can put it directly in VipsImage.c and avoid to include field.h

Reply:
yes but then the compiler doesn't check that the declarations match